ui/app/AppLayouts/Chat/views/MembersSelectorView.qml

```diff
@@ -108,6 +108,7 @@ MembersSelectorBase {
             }
 
             if (root.model.count === 0) {
+                root.suggestionsDialog.forceHide = true
```

Comment:
I like the idea of `forceHide` but I don't like the idea of imperative assignments (see comment above).

Could you please make it a binding?
`suggestionsDialog.forceHide: edit.text.startsWith(Constants.userLinkPrefix)`

Reply:
sorry, but I don't see any sense in this suggestion.
`forceHide` in this case must be set only if the chat link was correct (and if it is correct, we are showing a popup). 
This binding won't protect us from editing/corrupting/setting the invalid link

Reply:
Hm.. sorry, I was misled by the issue description:

> I think when we enter a chatkey (or status link) we should not show the "No results found" popup, because we know we're trying to chat with someone who isn't a contact yet.

Your solution makes more sense to me now although it will still show "No results found" for valid chat keys of already added contacts.

> > Only if I'll decide to put a contact, which is already in my contact list - I'll get no action.

> This seems correct. Although that should probably open the 1-on-1 chat right away.

Should we create a separate issue for that or you would be able to cover it within this PR?

Reply:
We already have a bug for opening 121 chat by link #8591 (I'm working on it)